relay/polkadot/src/lib.rs

```diff
@@ -2030,6 +2058,13 @@ pub mod migrations {
 		}
 	}
 
+	parameter_types! {
+		// This is used to limit max pools that migrates in the runtime upgrade.  This is set to
+		// existing_pool_count plus ~15 to also account for any new pools getting created before the
+		// migration is actually executed.
+		pub const MaxPoolsToMigrate: u32 = 250;
```

Comment:
@Ank4n @gpestana what happens if this number is wrong? And we don't migrate all of them?

Wouldn't it be more safe to set this to 1000?

Reply:
Yeah I can bump it up further, but should be fine either way.

We don't expect pools to be created that [frequently](https://polkadot.subscan.io/event?page=1&time_dimension=date&module=nominationpools&event_id=created). There are currently 229 pools in prod so 250 sounds hard to reach even in next couple of months. There is an [extrinsic](https://paritytech.github.io/polkadot-sdk/master/pallet_nomination_pools/pallet/struct.Pallet.html#method.migrate_pool_to_delegate_stake) in the improbable case if we do reach that number and some pools are unmigrated. 

This is also not really worth attacking as it only just delays migration of pool. Also pool creation is not free: the creator needs to bond 250 DOTs per pool.

Reply:
After https://github.com/pandres95/runtimes/pull/5, this changed to 500.

Closing conversation for now.